syntax = "proto3";

//[MAINTAINING CONNECTION]
message PingRequest
{
	string clientIp = 1;
};

message PingResponse
{
	string response = 1;
};

//[SERVICES]
service MaintainingApi {
    rpc Ping (PingRequest) returns (PingResponse) {}
}
